samples/apps/spring-petclinic/.minikube/deploy-app.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+NAMESPACE=${NAMESPACE:-my-postgresql}
+
+sed -e 's,REPLACE_PETCLINIC_APP_IMAGE,'${PETCLINIC_APP_IMAGE}',g' app-deployment.yaml | \
+sed -e 's,REPLACE_PETCLINIC_APP_DBINIT_IMAGE,'${PETCLINIC_APP_DBINIT_IMAGE}',g' | \
```

Comment:
the rest of the project uses `kustomize`, so if really needed we could use that instead of `sed`